src/pages/Integrations/SplunkSetup/SplunkSetupFinished.tsx

```diff
@@ -0,0 +1,26 @@
+import {
+    Button,
+    EmptyState,
+    EmptyStateBody,
+    EmptyStateIcon,
+    Title
+} from '@patternfly/react-core';
+import CheckCircleIcon from '@patternfly/react-icons/dist/js/icons/check-circle-icon';
```

Comment:
Could you use the short import ?

```suggestion
import { CheckCircleIcon } from '@patternfly/react-icons';
```

Reply:
Thank you! Fixed that and all other occurrences.